src/utils/features_preprocessing.py

```diff
@@ -223,6 +226,35 @@ def stack(self, tfidf, norm_indirect_features):
         return hstack((tfidf, norm_indirect_features))
 
 
```

Comment:
with open("../../data/tfidf.pickle", 'rb') as file:
            return pickle.load(file)
            
            Path - should be a parameter,
            better to have something like: self._tfidf = pickle.load(file)
            Thinks maybe you should remove load, and save if you are not using it in any place